### PR TITLE
#166151251 fix flake 8 errors

### PR DESCRIPTION
**Pull request by ceexon**
#### What does this PR do?
Fixes flake 8 syntactical errors

#### Description of Task to be completed?
Remove errors arising as a result of adding flake 8 linters ie, indentation errors, line length, and unused imports.

#### How should this be manually tested?
```
  $ pip install flake8 && flake8 wger
```
- set default editor linter to flake8

#### What are the relevant pivotal tracker stories?
#166151251

### CODE REVIEW

File: tasks.py [hunk 1]
```diff
@@ -86,7 +97,9 @@ def bootstrap_wger(context,
     if settings_path is None:
         settings_path = get_user_config_path('wger', 'settings.py')
     if not os.path.exists(settings_path):
-        create_settings(context, settings_path=settings_path, database_path=database_path, url=url)
+        create_settings(
```
Comment (wekesa931) [inline tasks.py]: Consider changing this to `create-settings` as it is deprecated

**Comment by wangonya** [comment]
Kindly edit the PR description on installing flake8 to be `pip install flake8` instead of `pip install flake 8`

**Review by wangonya — CHANGES_REQUESTED** [review]
Kindly edit the PR description on installing flake8 to be `pip install flake8` instead of `pip install flake 8`

**Review by jmusila — CHANGES_REQUESTED** [review]
I think you can add the command to run `flake8` from the terminal after installation.

**Review by wangonya — APPROVED** [review]

**Review by jmusila — APPROVED** [review]

**Review by wekesa931 — APPROVED** [review]

**Review by willeswa — APPROVED** [review]
LGTM

**Comment by caveinn** [comment]
@kburudi  kindly check on the dropping coverage

**Pull request merged**